addons/default/visiosoft/advs-module/resources/views/list/partials/ads.twig

```diff
@@ -5,16 +5,13 @@
     <table class="w-100 text-center">
         <thead>
         <tr class="text-white">
-            <th class="text-nowrap" colspan="2">{{ trans('visiosoft.module.advs::field.ad_title') }}</th>
+            <th class="text-nowrap" colspan="3">{{ trans('visiosoft.module.advs::field.ad_title') }}</th>
             {% if listLocation %}
                 <th class="text-nowrap">
                     {{ trans('visiosoft.module.advs::field.city.name') }}
                     / {{ trans('visiosoft.module.advs::field.district.name') }}
                 </th>
             {% endif %}
-            {% if setting_value('visiosoft.module.advs::market_place') %}
-                <th class="text-nowrap">{{ trans('visiosoft.module.advs::field.listing_date') }}</th>
-            {% endif %}
```

Comment:
why remove the header?

Reply:
@diashalabi #3835 -12 check Gitlab issue